src/protojl/tensorboard/plugins/custom_scalar/tensorboard/layout_pb.jl

```diff
@@ -1,5 +1,5 @@
-# Autogenerated using ProtoBuf.jl v1.0.11 on 2023-06-19T18:18:24.964
-# original file: /home/lior/TensorBoardLogger.jl/gen/proto/tensorboard/plugins/custom_scalar/layout.proto (proto3 syntax)
+# Autogenerated using ProtoBuf.jl v1.0.11 on 2023-08-09T10:18:18.779
+# original file: proto/tensorboard/plugins/custom_scalar/tensorboard/layout.proto (proto3 syntax)
```

Comment:
Here it's unclear that this file came from the `gen` folder, and the path seems inconsistent?